bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/transactions/TransactionResourceManager.java

```diff
@@ -96,6 +119,57 @@ public static TransactionResourceManager getInstance() {
         return transactionResourceManager;
     }
 
+    /**
+     * This method adds configs for transaction log path and transaction log name.
+     *
+     */
+    private void addLogConfigToRegistry() {
+        // Path to log directory given for atomikos log path property should exist.
+        // Therefore create a transaction_log_dir in root directory if not exists.
+        final Path projectRoot = findProjectRoot(Paths.get(System.getProperty("user.dir")));
+        if (projectRoot != null) {
+            String logPath = projectRoot.toAbsolutePath().toString() + "/transaction_log_dir";
```

Comment:
Is it possible for your to define the log directory as well? if he as not set it, we will use this name as the default one.

Reply:
fixed